README.md

```diff
@@ -5,7 +5,9 @@
 Layout engine for single-spa applications
 
 ## Project Goal
+
```

Comment:
First draft of this I did in github's code editor so the prettier hook didn't run. This one I did locally so the prettier hook ran. 

The spacing and `'` -> `"` changes are from prettier